web-app/src/components/mainLayout/AvatarMenu.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="row col-4 justify-end">
+    <q-btn round class="q-ma-sm">
+      <q-avatar :size="$q.screen.lt.sm ? '40px' : '50px'">
+        <img src="https://cdn.quasar.dev/img/boy-avatar.png" />
+        <!-- TODO user avatar -->
+      </q-avatar>
```

Comment:
this component is already made and ready to use

Reply:
what do you mean ?

Reply:
nvm found it